libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -132,7 +132,7 @@ proc send*(
   timeout: Duration = DefaultSendTimeout) {.async.} =
   logScope:
     peer = p.id
-    msg = shortLog(msg)
+    rpcMsg = shortLog(msg)
```

Comment:
I believe there might be many places in the codebase where `msg` is used... it's not an issue to change those, but it seems like a very big pitfall to have a reserved word in `chronicles` and not warn about it when used as a field, imo chronicles should warn about it.

Reply:
myeah, half of our code deals with`msg`:s, sad to not be able to use it

Reply:
This PR fixes a problem which we currently have, `nim-libp2p` producing incorrect json log lines.